package.json

```diff
@@ -36,6 +38,7 @@
     "azure-arm-authorization": "4.1.0",
     "azure-arm-automation": "2.0.0-preview",
     "azure-arm-batch": "2.2.0",
+    "azure-arm-batchai": "1.0.0",
```

Comment:
Please don't add "azure-arm-batchai" right now over here. I will add this later when we ship the rollup package. This will cause the CI to fail as it will try to install the package from npm and this package does not exist yet.

Reply:
removed